### DISCUSSION TITLE
[RG350] cant start the game with 1.3.0 version

### DISCUSSION THREAD

**Discussion by pascal4satu** [diasurgical/DevilutionX#4214] [Q&A]
hi diasurgical admin, the devilution v1.30 can't start almost the time. Just flash a black screen after i start the game and come back to home page in my RG350 machine. But with v1.10 the game was  worked fine. My RG350 firmware version is HDMI Patch version 1.1. is there the way to fix this ? but sometime the game can start. I love this version (v1.30) because can run the hellfire version.

**Comment by qndel** [comment]
We are pretty close to releasing 1.4.0, maybe it will fix your problem :)

**Reply by pascal4satu** [reply]
thank you for ur info @qndel . i hope it will be released soon. :)

**Comment by AJenbo** [comment]
It's probably going to take a few years before we reach version 1.10 or even 1.30 :shrug:

Hellfire support was added in 1.2

1.3.0 requires OpenDingux Beta so you need to update your firmware to run it on RG350. 1.4.0 isn't changing that afaik. So you should probably grab 1.2.1 if you are not interested in upgrading your firmware.

**Comment by john-parton** [comment]
As a user of the RG350M, I can highly recommend upgrading to the beta firmware. The ADAM image makes the whole process a lot easier. Comes preloaded with stuff. https://github.com/eduardofilo/RG350_adam_image
